src/shared/components/palette/complex/write-complex-field.html

```diff
@@ -1,15 +1,21 @@
 <div class="form-group" [id]="id()">
 
   <h3 *ngIf="renderLabel" class="heading-small heading-17">{{caseField | ccdFieldLabel}}</h3>
-
-  <ccd-field-write
-    *ngFor="let field of caseField | ccdFieldsFilter:true"
-    ccdConditionalShow
-    ccdLabelSubstitutor
-    [caseField]="field"
-    [formGroup]="complexGroup"
-    [registerControl]="buildControlRegistrer(field)"
-    [idPrefix]="buildIdPrefix(field)">
-  </ccd-field-write>
-
+  <ng-container *ngFor="let field of caseField | ccdFieldsFilter:true">
+    <ng-container [ngSwitch]="field | ccdIsReadOnly">
+      <ccd-field-read *ngSwitchCase="true"
+                      [caseField]="field"
```

Comment:
@rafalkalita don't we need `ccdConditionalShow` on the `ccd-field-read`? Without this show conditions might not work?

Reply:
Done as a part of show and hide RDM-3940